qa/tasks/vstart_runner.py

```diff
@@ -352,6 +352,23 @@ def sudo_write_file(self, path, data, **kwargs):
         """
         self.write_file(path, data, sudo=True, **kwargs)
 
+    def write_temp_file(self, **kwargs):
+        """
+        This is a convenience method that call write_file() but without sudo.
+        Reason for excluding sudo is that files in /tmp can not be written by
+        any user (including the root user) besides the file owner.
+
```

Comment:
Huh. I guess this is some of the new "hardening" patches, like this one:

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=30aba6656f61ed44cba445a3c0d38b296fa9e8f5

Sigh. Not sure I'm a fan, but I guess that ship has sailed.

Reply:
Yes, that's the exact commit.